### PR TITLE
Fix test by changing the assigned worker set to disable

### PR DESCRIPTION
**Pull request by NagaPradeepGajula-osi**
[#1250] https://osi-cwds.atlassian.net/browse/INT-1250

### CODE REVIEW

File: spec/screening/screening_info_spec.rb [hunk 1]
```diff
@@ -81,11 +75,10 @@
       expect(page).to have_field('Title/Name of Screening',
                                  with: screen_info_init[:title])
       expect(page).to have_field('Assigned Social Worker',
-                                 with: screen_info_init[:worker])
-      # Need test for how date is displayed once date picker is complete
+                                 disabled: true)
+    # Need test for how date is displayed once date picker is complete
```
Comment (VinhOSI) [inline spec/screening/screening_info_spec.rb]: why?

File: spec/screening/screening_info_spec.rb [hunk 2]
```diff
@@ -105,11 +98,10 @@
       expect(page).to have_field('Title/Name of Screening',
                                  with: screen_info_init[:title])
       expect(page).to have_field('Assigned Social Worker',
-                                 with: screen_info_chng1[:worker])
-      # Need test for how date is displayed once date picker is complete
+                                 disabled: true)
+    # Need test for how date is displayed once date picker is complete
```
Comment (VinhOSI) [inline spec/screening/screening_info_spec.rb]: why, again.

File: spec/screening/screening_info_spec.rb [hunk 3]
```diff
@@ -5,25 +5,22 @@
   before do
     visit '/'
     login_user
-    click_link 'Start Screening'
+    click_button 'Start Screening'
   end
 
   screen_info_init = {
-    title: 'Child @bandonment1',
-    worker: 'Jane $ocialworker',
```
Comment (VinhOSI) [inline spec/screening/screening_info_spec.rb]: Why didn't you just change this to "Assigned Social Worker" and kept everything else the same.

File: spec/screening/screening_info_spec.rb (hunk 1 above)
Comment (NagaPradeepGajula-osi) [reply spec/screening/screening_info_spec.rb]: In the current system, this field is disabled based on the login right away when you start screening. So is the change to expect if its there and is disabled.

File: spec/screening/screening_info_spec.rb (hunk 2 above)
Comment (NagaPradeepGajula-osi) [reply spec/screening/screening_info_spec.rb]: same, again.

File: spec/screening/screening_info_spec.rb (hunk 1 above)
Comment (VinhOSI) [reply spec/screening/screening_info_spec.rb]: Sorry. I meant to ask, why you are removing a space in the comment. Now it's not aligned correctly.

File: spec/screening/screening_info_spec.rb (hunk 3 above)
Comment (NagaPradeepGajula-osi) [reply spec/screening/screening_info_spec.rb]: In the current system, this field is disabled and you will be not able to fill_in the field and also you can't expect what will be in there even if it is auto populated.

Comment (NagaPradeepGajula-osi) [reply spec/screening/screening_info_spec.rb]: And for the title part one cannot insert any special characters/numerics in title column. So is the change.

File: spec/screening/screening_info_spec.rb (hunk 1 above)
Comment (NagaPradeepGajula-osi) [reply spec/screening/screening_info_spec.rb]: oh I think that might be due to beautifier config on my mac. Did not notice that.

**Pull request merged**